pkg/scalers/liiklus/LiiklusService_grpc.pb.go

```diff
@@ -1,17 +1,17 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
 // - protoc-gen-go-grpc v1.2.0
```

Comment:
please remove this from the PR, it is not relevant and we are downgrading the protoc version